pkg/routeagent_driver/handlers/ovn/ovn_logical_routes.go

```diff
@@ -120,10 +127,19 @@ func buildLRPsFromSubnets(subnetsToAdd []string, nextHop string) []*nbdb.Logical
 	toAdd := []*nbdb.LogicalRouterPolicy{}
 
 	for _, subnet := range subnetsToAdd {
+		var match string
+		if isIPv6(subnet) {
```

Comment:
You can use function  from k8snet here

// IsIPv6 returns true if netIP is IPv6 (and false if it is IPv4, nil, or invalid).
func IsIPv6(netIP net.IP) bool {
	return IPFamilyOf(netIP) == IPv6
}

// IsIPv6String returns true if ip contains a single IPv6 address and nothing else. It
// returns false if ip is an empty string, an IPv4 address, or anything else that is not a
// single IPv6 address.
func IsIPv6String(ip string) bool {
	return IPFamilyOfString(ip) == IPv6
}

Reply:
Use ipfamily